test/exampleGeneratorTests.ts

```diff
@@ -80,6 +80,7 @@ describe("test generate example", () => {
     ["sql", "package-pure-2020-02-preview"],
     ["signalr", "package-2020-05-01"],
     ["eventgrid", "package-2020-06"],
+    ["agrifood", "package-preview-2023-06"]
```

Comment:
Just curious: why we are adding `agrifood` samples in this PR?

Reply:
It includes a `pattern` usage in a parameter for the operation present within. Originally, when I removed the `mockerjs` dependency, these tests did **not** fail because none of the specs being exercised included a parameter that utilized the `pattern` property.

Now it does, so if the generation actually **failed** we'd know.